test/forked/rails_view_tracker_stack_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require File.expand_path("../rails_test_helper", File.dirname(__FILE__))
+
+class RailsWithoutConfigStackTest < Minitest::Test
+  def setup
+    super
+    setup_server
+  end
+
+  def teardown
+    super
+    shutdown_server
+  end
+
+  test "check view tracker" do
```

Comment:
This is great. One thought here and probably not very important. Could use use capybara to hit a remote server here? I've done this before:

```ruby
Capybara.run_server = false
Capybara.app_host = http://localhost:9999/
```

Reply:
interesting, I hadn't thought of that, it would be nice to use capybara visit and asset helpers vs CURL.

I can give it a go

Reply:
Obviously not so important but i had done it before in a project so figure would mention.

Reply:
hmm tried several things, basically rack-test driver doesn't support it then hit various issues when trying to use selenium with `run_server = false`... seems promising but requires some tweaking